rust/repo/cookiecutter.json

```diff
@@ -1,5 +1,6 @@
 {
   "repo_name": "repository",
+  "owners": "{{ cookiecutter.github_username }} @jcape",
```

Comment:
:x: Not sure if something is missing, but this doesn't seem to work for me.

Perhaps adding a local extension with something like
```python
def github_username():
    output = subprocess.run(["gh", "api", "user"], capture_output=True)
    if output.returncode != 0:
        return ""

    user = json.loads(output.stdout)
    return user["login"]
```

Reply:
Hmm, that could work, I was using [`~/.cookiecutterrc`](https://cookiecutter.readthedocs.io/en/stable/advanced/user_config.html)